exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/json/StoredConfigurationGsonSerializerTest.java

```diff
@@ -30,6 +32,35 @@
 
 class StoredConfigurationGsonSerializerTest {
 
+  private static final String CONFIG_EXAMPLE = "{\t\n"
+      + "\"previous_cfg_hash\": \"0000000000000000000000000000000000000000000000000000000000000000"
+      + "\",\t\n"
```

Comment:
Are those `\t` tabs? Are they needed?

Reply:
Intellij IDEA paste of Json file. Could be removed.